core/controllers/voice_artist.py

```diff
@@ -38,15 +39,42 @@ class AudioUploadHandler(base.BaseHandler):
     to the local datastore in dev).
     """
 
+    URL_PATH_ARGS_SCHEMAS = {
+        'exploration_id': {
+            'schema': {
+                'type': 'basestring',
+                'validators': [{
+                    'id': 'is_regex_matched',
+                    'regex_pattern': constants.ENTITY_ID_REGEX
+                }]
+            }
+        }
+    }
+
+    HANDLER_ARGS_SCHEMAS = {
+        'POST': {
+            'raw_audio_file': {
+                'schema': {
+                    'type': 'basestring'
+                 }
+             },
+             'filename': {
+                 'schema': {
+                    'type': 'basestring'
+                 }
+             }
```

Comment:
Look at the code of this controller, there are many other validations done on the file and its name, do you think some of these can be introduced here?

Reply:
Yep, basically try to replicate the checks from the code if possible.